kinetic/distribution.yaml

```diff
@@ -2574,6 +2574,17 @@ repositories:
       url: https://github.com/davidfischinger/haf_grasping.git
       version: kinetic
     status: maintained
+  hebiros:
+    doc:
+      type: git
+      url: https://github.com/HebiRobotics/HEBI-ROS.git
+      version: master
+    release:
```

Comment:
If you only want hte package to be documented, please do not add the release section. If you want to release a package you will need to [run bloom](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) and use a separate release repository